tests/api/test_api_charts.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+import requests
+
+
+@pytest.mark.parametrize(
+    "url,status_code",
+    [
+        pytest.param(
+            "http://localhost:5000/generate-charts/60.31062731740045/-100.06347656250001/tx_max/ann",
```

Comment:
Can we make the `http://localhost:5000`  configurable, for example, through an environment variable, but with http://localhost:5000 as default value.  It will then be easy to run the same tests against PROD/DEMO/QA instances

Reply:
Yes, good idea!